test/integration/modules/ingredients/IngredientResolver.spec.ts

```diff
@@ -51,3 +61,26 @@ describe("QUERY ingredient suggestions", () => {
     expect(result.data.ingredientSuggestions.length).toBeLessThanOrEqual(count);
   });
 });
+
+describe("QUERY popular ingredients", () => {
+  it("should return ingredients sorted by search count in descending order", async () => {
+    const result = await query({ query: popularIngredientsQuery });
+
+    const ingredients = result.data.popularIngredients;
+    for (let i = 1; i < ingredients.length; i++) {
+      expect(ingredients[i].searchCount).toBeLessThanOrEqual(ingredients[i - 1].searchCount);
```

Comment:
Vergleich das doch mit `toMatchObject` gegen eine korrekt sortierte Liste

Reply:
Da wüsste ich jetzt leider nicht, wie ich das machen sollte 😅 Dafür müsste ich dann ja immer wissen, was in der Test-Datenbank drin ist. So ist's unabhängig von der Datenbank und überprüft trotzdem, ob die Objekte richtig sortiert sind

Reply:
So wie du es jetzt hast du die gefahr das ein Test falsch ist und dir evtl. auch ein false-positive liefert. 
Du befüllst doch deine Datenbank mit testdaten und gegen diese kannst du prüfen.